doc/news/changes/incompatibilities/20190819PeterMunch

```diff
@@ -0,0 +1,3 @@
+Improvement: Rename parallel::Triangulation to parallel::TriangulationBase.
```

Comment:
Please be a bit more verbose and avoid "improvement" because it is not really that from a user perspective. It much better describes the intent of the class - some kind of interface class. I would write:
```
Changed: The parallel::Triangulation class has been renamed to parallel::TriangulationBase
to better reflect its purpose of a base to a series of parallel triangulations such as
parallel::distributed::Triangulation or parallel::shared::Triangulation.
```

Reply:
@kronbichler: done!